modules/aioseop_opengraph.php

```diff
@@ -1,1832 +1,1874 @@
-			}
-
-			if ( $author_img = $this->get_the_image_by_author( $p ) ) {
-				$image['author'] = $author_img;
-			}
-			$image  = array_flip( $img );
-			$images = array();
-			if ( ! empty( $image ) ) {
-				foreach ( $image as $k => $v ) {
-					$images[ $v ] = '<img height=150 src="' . $v . '">';
-				}
-			}
-
-			return array( $image, $images );
-		}
-
-		function get_the_image_by_author( $options = null, $p = null ) {
-			if ( $p === null ) {
-				global $post;
-			} else {
-				$post = $p;
-			}
-			if ( ! empty( $post ) && ! empty( $post->post_author ) ) {
-				$matches    = array();
-				$get_avatar = get_avatar( $post->post_author, 300 );
-				if ( preg_match( "/src='(.*?)'/i", $get_avatar, $matches ) ) {
-					return $matches[1];
-				}
-			}
-
-			return false;
-		}
-
-		function get_the_image( $options = null, $p = null ) {
-			$meta_key = $this->options['aiosp_opengraph_meta_key'];
-
-			return parent::get_the_image( array( 'meta_key' => $meta_key ), $p );
-		}
-
-		function get_the_image_by_default( $args = array() ) {
-			return $this->options['aiosp_opengraph_dimg'];
-		}
-
-		function settings_update() {
-
-		}
-
-		/**
-		 * Admin Enqueue Scripts
-		 *
-		 * Add hook in \All_in_One_SEO_Pack_Module::enqueue_metabox_scripts - Bails adding hook if not on target valid screen.
-		 * Add hook in \All_in_One_SEO_Pack_Module::add_page_hooks - Function itself is hooked based on the screen_id/page.
-		 *
-		 * @since 2.9.2
-		 *
-		 * @see 'admin_enqueue_scripts' hook
-		 * @link https://developer.wordpress.org/reference/hooks/admin_enqueue_scripts/
-		 *
-		 * @param string $hook_suffix
-		 */
-		public function admin_enqueue_scripts( $hook_suffix ) {
-			wp_enqueue_script(
-				'aioseop-opengraph-script',
-				AIOSEOP_PLUGIN_URL . 'js/modules/aioseop_opengraph.js',
-				array(),
-				AIOSEOP_VERSION
-			);
-
-			// Dev note: If certain JS files need to be restricted to select screens, then follow concept
-			// used in `All_in_One_SEO_Pack::admin_enqueue_scripts()` (v2.9.1); which uses the `$hook_suffix`
-			// and a switch-case. This also helps prevent unnessecarily processing localized data when it isn't needed.
-			parent::admin_enqueue_scripts( $hook_suffix );
-		}
-
-		/**
-		 * Enqueue our file upload scripts and styles.
-		 * @param $hook
-		 */
-		function og_admin_enqueue_scripts( $hook ) {
-
-			if ( 'all-in-one-seo_page_aiosp_opengraph' != $hook && 'term.php' != $hook ) {
-				// Only enqueue if we're on the social module settings page.
-				return;
-			}
-
-			wp_enqueue_script( 'media-upload' );
-			wp_enqueue_script( 'thickbox' );
-			wp_enqueue_style( 'thickbox' );
-			wp_enqueue_media();
-		}
-
-		function save_tax_data( $term_id, $tt_id, $taxonomy ) {
-			static $update = false;
-			if ( $update ) {
-				return;
-			}
-			if ( $this->locations !== null ) {
-				foreach ( $this->locations as $k => $v ) {
-					if ( isset( $v['type'] ) && ( $v
```

Comment:
Nice work updating the variables and description. Looks good.

The `since` tag might have been misplaced though. Normally it's suppose to be...

```
/**
 * Summary.
 *
 * @since x.x.x
 *
 * @param type $var Description.
 * @param type $var Optional. Description. Default.
 * @return type Description.
 */
```
https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/php/#1-functions-class-methods

Reply:
done